src/Microsoft.AspNetCore.SignalR.Specification.Tests/Microsoft.AspNetCore.SignalR.Specification.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Tests for users to verify their own implementations of SignalR types</Description>
+    <DeveloperBuildTestTfms>netcoreapp2.2</DeveloperBuildTestTfms>
```

Comment:
Change this to use `<TargetFramework>` like a normal class library project. Also, consider making this netstandard2.0.